core/optaplanner-constraint-streams/src/main/java/org/optaplanner/constraint/streams/bavet/tri/BavetAbstractTriConstraintStream.java

```diff
@@ -79,20 +78,12 @@ public BavetAbstractTriConstraintStream<Solution_, A, B, C> filter(TriPredicate<
     // ************************************************************************
 
     @Override
-    public <D> QuadConstraintStream<A, B, C, D> actuallyJoin(UniConstraintStream<D> otherStream,
-            DefaultQuadJoiner<A, B, C, D>... joiners) {
+    @SafeVarargs
+    public final <D> QuadConstraintStream<A, B, C, D> join(UniConstraintStream<D> otherStream,
```

Comment:
StreamHelpers removed (mostly replaced by JoinerComber), so the join method is now directly here.